lib/core/services/alfresco-api.service.ts

```diff
@@ -49,25 +53,54 @@ export class AlfrescoApiService {
     }
 
     async load() {
-        await this.appConfig.load().then(() => {
-            this.storageService.prefix = this.appConfig.get<string>(AppConfigValues.STORAGE_PREFIX, '');
-            this.initAlfrescoApi();
-            this.alfrescoApiInitialized.next(true);
+        await this.appConfig.load()
```

Comment:
We are about to completely deprecate this mess what we have in this service and provide a clean solution. I'm not sure it is a good idea to add new things here.